addons/sale/tests/test_credit_limit.py

```diff
@@ -29,6 +29,41 @@ def setUpClass(cls):
             'currency_id': buck_currency.id,
         })
 
+    def test_credit_limit_multi_company(self):
+        # multi-company setup
+        company2 = self.company_data_2['company']
+        self.assertNotEqual(self.env.company.id, company2.id)
+
+        # Activate the Credit Limit feature and set a value for partner_a.
+        self.env.company.account_use_credit_limit = True
+        self.partner_a.credit_limit = 1000.0
+
+        # Create and confirm a SO for another company
+        sale_order = company2.env['sale.order'].create({
+            'company_id': company2.id,
+            'partner_id': self.partner_a.id,
+            'partner_invoice_id': self.partner_a.id,
+            'partner_shipping_id': self.partner_a.id,
+            'pricelist_id': self.company_data_2['default_pricelist'].id,
+            'order_line': [Command.create({
+                'name': self.company_data_2['product_order_no'].name,
+                'product_id': self.company_data_2['product_order_no'].id,
+                'product_uom_qty': 1,
+                'product_uom': self.company_data_2['product_order_no'].uom_id.id,
+                'price_unit': 1000.0,
+                'tax_id': False,
+            })]
```

Comment:
```suggestion
            'order_line': [Command.create({
                'product_id': self.company_data_2['product_order_no'].id,
            })]
```

Reply:
I left the 'price_unit' in. I think it makes it easier to follow read the tests.
Is it okay?